package.json

```diff
@@ -13,11 +13,12 @@
     "babel-plugin-transform-es2015-destructuring": "^6.6.5",
     "babel-preset-es2015": "^6.6.0",
     "browser-sync": "^2.12.10",
-    "browserify": "^13.0.1"
+    "browserify": "^13.0.1",
+    "marked": "^0.3.5"
```

Comment:
`marked` is deprecated. It hasn't seen a commit in a really, really long time.

[Try `remarkable` instead](https://github.com/jonschlinkert/remarkable)!